txlive.phantomjs

```diff
@@ -5,26 +5,29 @@ var address = args[1];
 
 page.onInitialized = function() {
   page.evaluate(function() {
-    if (!window.Transifex || !window.Transifex.live)
-      document.write('<script src="//cdn.transifex.com/live.js"></script>');
+    if (!window.Transifex || !window.Transifex.live) {
+      //document.write('<script src="//cdn.transifex.com/live.js"></script>');
+      document.write('<script src="//s3.amazonaws.com/tx-live-dev/live.js"></script>');
```

Comment:
IMPORTANT: This needs to be reverted as soon as live.js V42 (with xpath support) is deployed.